src/index.html

```diff
@@ -1,22 +1,463 @@
+<!-- #region HEAD -->
 <!doctype html>
-<html lang="en">
+<html
+  lang="en"
+  class="page"
+>
   <head>
     <meta charset="UTF-8" />
     <meta
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
-    <title>Landing Page</title>
+    <title>Bang & Olufsen</title>
+    <link
+      rel="icon"
+      href="../src/images/logo-phone-320.png"
+      type="image/png"
+    />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Manrope:wght@200..800&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
-      href="./styles/main.scss"
+      href="./styles/style.scss"
     />
     <script
       src="scripts/main.js"
       defer
     ></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <!-- #endregion -->
+  <body class="page__body">
+    <header class="header">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-bar">
+            <a
+              href="#"
+              class="top-bar__logo-link"
+            >
+              <img
+                src="images/logo.png"
+                alt="b&o logo"
+                class="top-bar__logo"
+              />
+            </a>
+            <div class="top-bar__icons">
+              <a
+                href="tel:+123455555555"
+                class="icon icon--phone"
+                aria-label="Call us"
+              ></a>
+              <a
+                href="#menu"
+                class="icon icon--menu"
+              ></a>
+            </div>
+          </div>
+          <div class="header__bottom">
+            <h1 class="header__title">
+              Your senses.
+              <br />
+              Set free.
+            </h1>
+          </div>
+        </div>
+      </div>
+    </header>
+
+    <aside
```

Comment:
This gray background does not exist on Figma, giving an strange look to the menu.

![image](https://github.com/user-attachments/assets/cca16bba-e8dc-4fba-ad78-6bb1df519557)


on Figma:

![image](https://github.com/user-attachments/assets/1a1fd185-8007-4b1f-ad0c-fbd6126a1d0b)